Gruntfile.js

```diff
@@ -392,6 +392,29 @@ module.exports = function(grunt) {
                   grunt.config('meta.banner'), grunt.config('meta.cssFileBanner'));
   });
 
+  grunt.registerTask('makeVersionsMappingFile', function () {
+    var done = this.async();
```

Comment:
I believe you can just pass done as an argument to the callback defining the task